tests/cases.py

```diff
@@ -1639,25 +1649,33 @@ def post_instantiation_hook(self) -> None:  # noqa: D102
 
     def _get_input(
         self,
-        inverse: bool = False,
+        target: Target,
     ) -> Tuple[torch.LongTensor, torch.FloatTensor, Optional[torch.BoolTensor]]:
         # Get batch
         hrt_batch = self.factory.mapped_triples[: self.batch_size].to(self.model.device)
 
         # Compute scores
-        if inverse:
+        if target == LABEL_HEAD:
             scores = self.model.score_h(rt_batch=hrt_batch[:, 1:])
-        else:
+        elif target == LABEL_RELATION:
+            scores = self.model.score_r(ht_batch=hrt_batch[:, [0, 2]])
```

Comment:
We could consider not using the model at all, and instead creating some mock scores without any model.